ci/Selenium/src/test/java/bfui/test/TestCreateJob.java

```diff
@@ -65,33 +69,73 @@ public void tearDown() {
 	@Test
 	@Info(importance = Importance.HIGH)
 	public void landsat_job_with_mask() throws Exception {
```

Comment:
These methods should follow `lowerCamelCase` naming conventions, not `snake_case`.

Reply:
Done